src/views/orb-view.ts

```diff
@@ -44,7 +44,11 @@ export interface IOrbViewSettings<N extends INodeBase, E extends IEdgeBase> {
 export type IOrbViewSettingsInit<N extends INodeBase, E extends IEdgeBase> = Omit<
   Partial<IOrbViewSettings<N, E>>,
   'render'
-> & { render?: Partial<IRendererSettingsInit> };
+> & { render?: Partial<IRendererSettingsInit> } & { 
+  instances: {
+    simulator?: ISimulator | (() => ISimulator);
+  };
```

Comment:
```suggestion
> & {
  render?: Partial<IRendererSettingsInit>,
  instances?: {
    simulator?: ISimulator | (() => ISimulator);
  };
```

I would say that you don't need to join them with `&` - just append to the `render` interface. Btw `instances` should be optional, if not users would need to always initiate an empty object even if they don't want to override the simulator instance.